AppCenter/AppCenterTests/Util/MSMockSecondService.m

```diff
@@ -0,0 +1,50 @@
+#import "MSMockSecondService.h"
+#import "MSChannelConfiguration.h"
+
+static NSString *const kMSServiceName = @"MSMockSecondService";
```

Comment:
Why don't you just use MSMockService by adding setters for serviceName and groupId in `init`?

Reply:
I would, but `serviceName` is class-level, so if I changed it then it would return the wrong value for the other one

Reply:
I can make this better by making a base class for MSMockService, which would allow easy creation of mock services with different names for scenarios like this one

Reply:
Then why don't you extend MSMockService class instead of writing all methods in the second class?

Reply:
Because class level methods can't be overridden, so it would require one base class and two derived classes

Reply:
which I can do

Reply:
You can create member methods and call this from class level methods. Then you just need to override methods that you created. Whenever you call class level methods, `sharedInstance` will always be ready to use.

Reply:
I thought something wrong there. `sharedInstance` is also a class level method. 😮 
Don't mind.